WordPress/WordPressUITests/Tests/EditorGutenbergTests.swift

```diff
@@ -27,14 +28,15 @@ class EditorGutenbergTests: XCTestCase {
         try super.tearDownWithError()
     }
 
+    let title = "Rich post title"
+    let content = "Some text, and more text"
+
     func testTextPostPublish() throws {
 
-        let title = "Text post title"
-        let content = "Text post content"
         try editorScreen
-            .dismissNotificationAlertIfNeeded(.accept)
             .enterTextInTitle(text: title)
             .addParagraphBlock(withText: content)
+            .verifyContentStructure(blocks: 1, words: 5, characters: 24)
```

Comment:
The value is from `let content`, what do you think if we do the word and character count programmatically in case the value of `content` changes?